src/Service/SearchEngineManager.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Skill;
+use App\Repository\SkillRepository;
+
+use App\Entity\Usergroup;
+use App\Repository\UsergroupRepository;
+
+use App\Entity\User;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Container\ContainerInterface;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\Extension\Core\Type\SearchType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\Intl\Intl;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+use App\Form\TagType;
+
+class SearchEngineManager {
+	private $manager;
+	private $formFactory;
+
+	public function __construct ( EntityManagerInterface $manager, ContainerInterface $container ) {
```

Comment:
Il ne faut pas passer le container dans un service, c'est une mauvaise pratique. 

```suggestion
	public function __construct ( EntityManagerInterface $manager, FormFactoryInterface $formFactory ) {
```

Reply:
Done ! 

Bon à savoir, cette erreur apparaît aussi dans 3 autres fichiers: 
- UserFileManager.php
- UsergroupFileManager.php
- UsergroupMembersManager.php

Reply:
Ah bien vu ! À corriger eventuellement les prochaines fois où on touchera à ces fichiers